src/Sulu/Bundle/TagBundle/Controller/TagController.php

```diff
@@ -356,4 +357,40 @@ protected function getData(Request $request)
     {
         return $request->request->all();
     }
+
+    /**
+     * Returns array of tags from escaped input string.
+     */
+    private function explodeTags(string $input): array
+    {
+        $tags = [];
+        $append = false;
+        foreach (explode(',', $input) as $tag) {
```

Comment:
I think we should be able to escape the comma in a different way, since `explode` also excepts more than one character. Would probably make the code aftewards a lot easier.

Reply:
https://github.com/sulu/sulu/pull/4977#discussion_r366756243